src/common/tokenMintKeys.ts

```diff
@@ -0,0 +1,9 @@
+import { NATIVE_MINT } from "@solana/spl-token"
+
+export const mainTokenMintKeys = {
+  SOL_NATIVE: NATIVE_MINT,
+  SOL: "So11111111111111111111111111111111111111112",
```

Comment:
These two are the same
```suggestion
  NATIVE_MINT,
```

Reply:
This will work for now but it's not flexible when it comes to different clusters, dev environments and other situations

Later on let's work on something longer term, either being able to locate them and figure out their names, or something else